source/ptreferential/tests/ptref_test.cpp

```diff
@@ -187,6 +187,14 @@ BOOST_AUTO_TEST_CASE(parser_method) {
     BOOST_CHECK_EQUAL(filters[0].args, std::vector<std::string>{"john"});
 }
 
+BOOST_AUTO_TEST_CASE(parser_method_has_disruption) {
```

Comment:
I think we should also test that we are able to get the vj that have disruptions no ?